content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Remote Optimization Improvements in DVC 1.0
+date: 2020-06-29
+description: |
+  An overview of how data synchronization to and from remote storage is optimized in DVC 1.0.
+picture: ''
+author: peter_rowlands
+---
+
+One of the key features provided by DVC is the ability to efficiently
+synchronize versioned datasets between a user's local machine and
+[remote storage](https://dvc.org/doc/command-reference/remote), and our users
+have frequently shown the need for us to optimize this process wherever
+possible. In addition to simply being frustrating for a user, slow runtimes for
+data synchronization operations will have a negative impact on how quickly new
+iterations of an ML project can be made.
+
+![](/uploads/images/2020-06-29/optimization_screenshot.png '=600')
+
+DVC 1.0 includes several performance improvements with regard to how DVC
+synchronizes data to and from remotes. In particular, for this release we
+focused on optimizing performance in situations where users with must regularly
+`dvc push` (relatively) small modifications made to large datasets versioned
+with DVC.
+
+Ideally, `dvc status -c` should always be a (relatively) fast operation, and the
```

Comment:
I would assume that people are not that familiar with DVC . It's better even to assume that they are not familiar at all and try to describe a higher level problem first - we have one million files. We want to sync it with a cloud - backup, share with the team, etc. There are a bunch of tools that can do that - rsync, aws sync, rcloud, etc ... They all need to calculate diff somehow - it's a crucial operation - to understand what exactly do we need to upload or download. 

Here a good point to "sell" with graphs "DVC is 20x faster than rsync", etc

Then it's probably good to go with generic tools approaches, naive approaches, etc. There are different possible ways of doing this (batch vs one-by-one) .. all of them have limitations in certain scenarios. Describe in what situations those approaches are slow.

After that go with interlude you did that describes DVC approach.

Reply:
latest draft has new sections with more general explanation about the problem, and with more explanation about the difference between DVC vs generic tools